development-process.md

```diff
@@ -0,0 +1,144 @@
+fix/close an issue (typically true), fill in this section with "fixes #123" or "closes #123".**
+When your PR is eventually merged, this text is parsed, and the issue will be automatically closed
+if "fixes #X"/"closes #X" is found.
+
+Before (or after) you submit the PR, add the "Needs Review" label to it.
+
+**TL;DR**
+1. Click "New Pull Request"
+1. Fill in (or confirm) the appropriate **base** and **compare** branches
+1. Fill in all the relevant sections in the PR template (especially "relevant tickets", which should
+   typically read "fixes #<issue_number>")
+1. Add the "Needs Review" label
+1. Submit and wait for comments/approval
+
+#### &#10146; Respond to comments until you get approval
+
+As comments come in and changes are requested, you can make any necessary changes and
+
+Reviewers should change the PR's label to "Waiting on author" when they are finished making comments
+and requesting changes. Similarly, when you are finished making changes based on that feedback and
+responding to comments, you should change the label back to "Needs Review".
+
+Right now, we consider a PR to be "approved" if the assigned reviewer on the PR has given you a +1
+(thumbs up). Use your own discretion about seeking out further approvals. That might be appropriate
+if there is someone very familiar with the code you're working on, but isn't the assigned reviewer.
+
+**TL;DR**
+- Make changes/respond to comments as needed and reset the PR label to "Needs Review"
+- PR is considered "approved" when you get a +1 (thumbs up) from the assigned reviewer
+
+#### &#10146; (As necessary) Rebase onto master and resolve conflicts when they occur
+
+As you work on your PR, changes may be made to master that cause would-be merge conflicts. The
+merge-ability of your branch will be indicated near the bottom of your PR page. If conflicts are detected,
+you should rebase your branch onto master, fix the conflicts, and push your branch. Here is a [rebasing
+guide maintained by edX](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request). The
+process looks something like this in the command line:
+
+    # Assuming (a) your issue branch is currently checked out...
+    git fetch && git rebase master
+    # Resolve merge conflicts, commit, repeat until done
+    # When merge conflicts are all resolved...
+    git push
+
+That message near the bottom of your PR page should have a green check mark and read "This branch has no
+conflicts with the base branch" when you have successfully resolved and pushed.
+
+
+### Step 3) Merging
+
+#### &#10146; "Squash" the commits on your branch
+
+You may have many incremental commits on your issue branch. In order to keep the git log for the master
+branch clean and readable, we "squash" all issue branch commits into one commit before merging. The commit
+message should be a past tense description of your entire PR, eg: "Updated the home page to match new design".
```

Comment:
Might be worth adding `Using the past tense makes it easier to write release notes.`